test/cypress/integration/user/dashboard_spec.js

```diff
@@ -1,25 +1,42 @@
 describe('Dashboard', () => {
-  const selfServiceUsers = require('../../../fixtures/config/self_service_user.json')
+  const userExternalId = '7d19aff33f8948deb97ed16b2912dcd3'
+  const gatewayAccountId = '666'
+  const serviceName = 'Test Service'
 
   beforeEach(() => {
-    cy.setCookie('session', Cypress.env('encryptedSessionCookie'))
-    cy.setCookie('gateway_account', Cypress.env('encryptedGatewayAccountCookie'))
+    cy.task('getCookies', {
```

Comment:
Cypress tasks are a way of calling node code during a test, and are defined in plugins/index.js